INSTALL.md

```diff
@@ -283,52 +292,52 @@ sudo systemctl restart redis
 
 Eventmachine has to be installed with `--with-cppflags=-I/usr/include/openssl/`. Simply run:
 
-```shell
+```
 bundle config build.eventmachine --with-cppflags=-I/usr/include/openssl/
 ```
 
 #### Config files
 
 Copy example config files from the examples folder:
 
-```shell
+```
 cp config/examples/* config/
 ```
 
 #### Bundle
 
 Run [Bundler](https://bundler.io/) to install all required Ruby gems:
 
-```shell
+```
 bundle install
 ```
 
 #### Node packages
 
 Install [Yarn](https://yarnpkg.com/):
 
-```bash
-brew install yarn
+```
+sudo dnf install yarnpkg
```

Comment:
I can't find this package. Maybe `npm install --global yarn` ?

Reply:
According to https://developer.fedoraproject.org/tech/languages/nodejs/nodejs.html it should be available since Fedora 29.

Reply:
Hmm, now I see it! :upside_down_face:  Don't know what went wrong the first time I tried :grimacing:

Reply:
I couldn't install the dependencies using the `yarnpkg` package in `dnf`. Don't remember the error, but I had to uninstall it and use the yarn package at `npm install --global yarn`. That worked. Fedora 36.